src/components/include/connection_handler/connection_handler.h

```diff
@@ -129,6 +129,15 @@ class ConnectionHandler {
  */
   virtual void SendEndService(uint32_t key, uint8_t service_type) = 0;
 
+  /**
+   * @brief Check is heartbeat monitoring started for specified connection key
+   * @param  connection_key pair of connection and session id
```

Comment:
@AKalinich-Luxoft Can not see pair in parameters list. Only one value `uint32_t`

Reply:
@AByzhynar thats because connection_key contains two encoded values - connection handle and session id. Encoding of this pair of values forms a 32-bit integer connection key. You can check implementation of PairFromKey/KeyFromPair functions for details.

Reply:
@AKalinich-Luxoft I understand it. But your description is wrong

Reply:
@AByzhynar ok then. Updated in https://github.com/AKalinich-Luxoft/sdl_core/pull/2/commits/d3bca99da653e13fdb1e7986d4ea6f09ca37ba11